include/swift/Option/Options.td

```diff
@@ -187,6 +187,9 @@ def swift_version : Separate<["-"], "swift-version">, Flags<[FrontendOption, Par
   HelpText<"Interpret input according to a specific Swift language version number">,
   MetaVarName<"<vers>">;
 
+def swift_runtime : Separator<["-"], "swift-runtime">, Flags<[FrontendOption]>,
+  HelpText<"Specify the target Swift runtime version">, MetaVarName<"<runtime>">;
```

Comment:
You don't really describe the expected values of this option.

`-swift-runtime-version` would be more consistent with `-swift-version`.